packages/core/src/browser/xhrProxy.spec.ts

```diff
@@ -209,4 +209,26 @@ describe('xhr proxy', () => {
       },
     })
   })
+
+  it('should track only the second request (the first request is canceled)', (done) => {
```

Comment:
Beside a unit test, it could be nice to have some E2E tests in `test/e2e/scenario/rum/resources.scenario.ts`. Because XHRs are mocked for unit tests, we might miss issues occuring in real browsers.